estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_offer_action" model="ir.actions.act_window">
+        <field name="name">Property Offers</field>
+        <field name="res_model">estate.property.offer</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <record id="estate_property_offer_view_form" model="ir.ui.view">
+        <field name="name">estate.property.offer.form</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="price"/>
+                        <field name="partner_id"/>
+                        <field name="state"/>
```

Comment:
Isn't `property_id` also required by the model?

Reply:
Despite the fact that the property_id field is required, and it is not include it in the views. The documentation states that Odoo knows which property is linked to as part of the magic of using the Odoo framework: sometimes things are defined implicitly. It states that when we create a record through a one2many field, the corresponding many2one is populated automatically for convenience, so I think that it is ok.

Reply:
You're right!

That's okay